int64_t is pulled in transitively on glibc
add the missing include so headers also builds on musl

--- a/library/public/video_frame.h
+++ b/library/public/video_frame.h
@@ -4,6 +4,8 @@
 #include "exception.h"
 #include "export.h"

+#include <cstdint>
+
 namespace f3d
 {
 /**
--- a/library/public/video_packet.h
+++ b/library/public/video_packet.h
@@ -4,6 +4,8 @@
 #include "exception.h"
 #include "export.h"

+#include <cstdint>
+
 namespace f3d
 {
 /**
